packages/manager/tools/dev-server/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "@ovh-ux/dev-server",
+  "version": "0.0.0",
+  "description": "Provide OVHcloud Manager dev server",
+  "keywords": [
+    "codename",
+    "manager",
+    "ovh"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/ovh/manager.git",
+    "directory": "packages/manager/tools/dev-server"
+  },
+  "license": "BSD-3-Clause",
+  "author": "OVH SAS",
+  "main": "src/index.js",
+  "bin": {
+    "dev-server": "bin/dev-server.js"
+  },
+  "dependencies": {
+    "commander": "^4.0.0",
+    "express": "^4.17.1",
+    "http-proxy-middleware": "^1.0.4"
+  },
+  "engines": {
+    "node": ">=8",
+    "yarn": ">=1.0.0"
+  }
```

Comment:
**nitpick:** I'm not sure that we specifically want to support these engines versions.

I would suggest to be aligned with the existing [prerequisites](https://github.com/ovh/manager/#prerequisites).

```suggestion
```